app/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
     }
     buildTypes {
         debug {
-            buildConfigField 'String', "ApiKey", Guardian_ApiKey
+            buildConfigField 'String', "ApiKey", '"61569e6e-7690-491d-b4fc-09ed040cca54"'
```

Comment:
Hey @Leviaran, you shouldn't be exposing your API key to the public. Please read the README.md file on how to setup the project.

Reply:
Done, thanks @NalediMadlopha. i will commit it, but i still dont understand how smaller chunk that i have to broke down.

Reply:
Alright, so you will probably agree with me that code review is fun but it can be a very tedious job to do when there are a lot of unrelated changes with too many files to review. 

If you look at the Files Changed tab above, you'll realize that you have made changes to 20 files. Some changes in these files are directly and indirectly related. 

What you'd want to do is only push files related to specific changes. For instance, your pull request is titled MVVM Pattern. I as a reviewer, have to assume that the intention of this  PR is to implement the MVVM pattern. However when I review the code you have implemented data-binding, changed the adapter, changed the models, added a datasource and so on. All these changes could be separated into their own pull requests (smaller chunks) instead of a huge PR that contains all these changes. 

This is just a suggestion, something to consider when you open you next PR.